docs/data/stashcache/install-cache.md

```diff
@@ -5,192 +5,221 @@ This document describes how to install a StashCache cache service.  This service
-By default this is `/stash`.
+By default this is `/mnt/stash`.
 We recommend using a separate file system for the cache directory,
 with at least 1 TB of storage available.
 
 !!! note
     The cache directory must be writable by the `xrootd:xrootd` user and group.
 
-The `stashcache-cache-server` provides a default configuration file,
-`/etc/xrootd/xrootd-stashcache-cache-server.cfg`, which must be customized for your cache.
+The `stash-cache` package provides default configuration files in `/etc/xrootd/xrootd-stash-cache.cfg` and `/etc/xrootd/config.d/`.
+Administrators may provide additional configuration by placing files in `/etc/xrootd/config.d/1*.cfg` (for files that need to be processed BEFORE the OSG configuration) or `/etc/xrootd/config.d/9*.cfg` (for files that need to be processed AFTER the OSG configuration).
 
-The most common lines to customize are:
+You _must_ configure every variable in `/etc/xrootd/10-common-site-local.cfg`.
 
-* `all.sitename YOUR_SITE_NAME`: The registered OSG resource name.  This **must** be changed.
-* `set cachedir = /stash`: The location of the cache directory for this service.
-* `*.trace`, `pss.setopt`: These control the logging verbosity of the cache.  The defaults are relatively high
-  in order to aid in debugging.  These lines can be commented out to reduce logging; however, if issues occur,
-  OSG support may ask you to re-enable them.
+The mandatory variables to configure are:
 
-The Authfile specifies which files and directories can be read,
-relative to the cache directory (`cachedir` in the main config).
+- `set rootdir = /mnt/stash`: the mounted filesystem path to export.  This document refers to this as `/mnt/stash`.
+- `set resourcename = YOUR_RESOURCE_NAME`: the resource name registered with the OSG.
 
-An example:
-
-```console
-root@host # cat /etc/xrootd/Authfile-noauth
-u * /user/ligo -rl / rl
-```
-
-This permits all users (`u *`) to read all directories (`/ rl`) _except_ those under `/user/ligo` (`/user/ligo -rl`);
-the `/user/ligo` directory should only be readable in authenticated setups.
-For more details, see the [XRootD security documentation](http://xrootd.org/doc/dev47/sec_config.htm#_Toc489606598).
 
 Configuring the Authenticated Cache (Optional)
 ----------------------------------------------
 
 The authenticated cache service is a separate instance of the StashCache cache service,
 and runs alongside the unauthenticated instance.
 You should make sure that the unauthenticated instance is functioning before setting up the authenticated instance.
-Before proceeding, make sure you have followed the [prerequisite steps](#installation-prerequisites-for-cache).
+Before proceeding, make sure you have followed the [prerequisite steps](#before-starting)).
 
-### Add Authfile for authenticated cache
 
-The Authfile for the authenticated cache is located in `/etc/xrootd/Authfile-auth`.
-This is a separate file from the non-authenticated cache Authfile.
+### Add a service certificate
```

Comment:
This needs to be moved out of the 'authenticated cache' section because adding a service cert is now required (for reporting).